source/ConfigSetup.h

```diff
@@ -31,6 +31,7 @@ EMP_BUILD_CONFIG(SymConfigBase,
     VALUE(PHYLOGENY, bool, 0, "Should the world keep track of host and symbiont phylogenies? (0 for no, 1 for yes)"),
     VALUE(NUM_PHYLO_BINS, size_t, 5, "How many bins should organisms be sepeated into if phylogeny is on?"),
     VALUE(NO_MUT_UPDATES, int, 0, "How many updates should be run after the end of UPDATES with all mutation turned off?"),
+    VALUE(SYM_EXCLUDE, bool, 0, "Do symbionts have a decreasing chance of getting into the host the more symbionts are already infecting it? (0 for no, 1 for yes)"),
```

Comment:
Let's include an alias so that old config files will still work for the new name, should just go right after the description string

Reply:
You could also then test it by not changing the tests :)